bitnami/redis/templates/metrics-svc.yaml

```diff
@@ -3,7 +3,7 @@ Copyright VMware, Inc.
 SPDX-License-Identifier: APACHE-2.0
 */}}
 
-{{- if .Values.metrics.enabled }}
+{{- if and .Values.metrics.enabled .Values.metrics.service.enabled }}
```

Comment:
I think we could use `.Values.metrics.serviceMonitor.enabled` WDYT? The new value won't be required.

Reply:
Hmm I think it would be nice to make it possible to retain the metrics `Service` without the `ServiceMonitor`. When not using the prometheus operator you can annotate the `Service` to have it scraped without using the `ServiceMonitor`:
```yaml
kind: Service
annotations:
  prometheus.io/port: "80"
  prometheus.io/scrape: "true"
```

Pretty sure those might still be around in the wild

Reply:
Ok, LGTM